CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 - [timePicker] Fix display on small devices
+- [Checkbox] Reverse label and input and more improvements
```

Comment:
`Reverse label and input and made it configurable`

Reply:
Done